designer/client/components/ComponentCreate/__tests__/ComponentCreateList.jest.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import { initI18n } from "../../../i18n";
+import { ComponentCreateList } from "../ComponentCreateList";
+
+initI18n();
+
+describe("ComponentCreateList", () => {
```

Comment:
I think we had agreed snapshots were out and instead we'd be using `expect(getByText("Delete list")).toBeInTheDocument();`? Could you please confirm with Enda?

Reply:
i think there snapshot is better for this as there are 25 long texts to be checked if getByText used